### DISCUSSION TITLE
Daily saves cleanup

### DISCUSSION THREAD

**Discussion by rdalbuquerque** [mbround18/valheim-docker#665] [Ideas]
Hello!

Is there a way to configure daily saves clean up? it's causing my storage to go full and I honestely don't care about keeping all valheim days. Would be really nice to have an environment variable to configure daily saves cleanup!

**Comment by spannerman79** [comment]
Answered in https://github.com/mbround18/valheim-docker/issues/664#issuecomment-1428857304
